openpype/modules/clockify/ftrack/server/action_clockify_sync_server.py

```diff
@@ -48,13 +48,19 @@ def discover(self, session, entities, event):
         return True
 
     def launch(self, session, entities, event):
+        self.clockapi.set_api()
+        workspace_id = self.clockapi.workspace_id
+        user_id = self.clockapi.workspace_id
         if self.clockapi.workspace_id is None:
             return {
                 "success": False,
                 "message": "Clockify Workspace or API key are not set!"
             }
 
-        if self.clockapi.validate_workspace_perm() is False:
+        if (
+            self.clockapi.validate_workspace_permissions(workspace_id, user_id)
+            is False
+        ):
```

Comment:
Variable `user_id` is using `workspace_id`, but bot variables `workspace_id` and `user_id` are defined only because of this line which looks confusing especially when `user_id` is overriden at line 70 and `workspace_id` is not used at line 54 please change it to this:
```suggestion
        if not self.clockapi.validate_workspace_permissions(
            self.clockapi.workspace_id, self.clockapi.user_id
        ):
```